redis-cluster/README.md

```diff
@@ -1,121 +1,103 @@
-# redis cluster base manifests
+# Redis Cluster
 
-This directory contains a redis base for setting up a redis cluster that is highly available and horizontally scales.
-It will spin up a [redis cluster](https://redis.io/docs/latest/operate/oss_and_stack/management/scaling/) with a number of
-replicas allowing for nodes to rotate with no downtime. 
+## TL;DR
 
-### Getting started
-The simplest way to create a redis cluster is to reference the base in a kustomization file. 
-```yaml
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+- [Is this Redis right for you?](#flavour)
+- [Example kustomization file deploying Redis](example)
+- [Test it manually using redis-cli](#usage)
+- [Grafana dashboards](#metrics)
 
-resources:
-- ../base
-```
+## Source
+
+This manifest is basing on multiple different manifests found over the internet.
+
+## Flavour
+
+This manifest is for clustered Redis.
+
+Before using those manifests, consider whether clustered is suitable for your use.
+There are following alternatives:
+
+|                                                             | High availability                                            | Mulitple databases | Sharding | Amount of nodes        | Remarks                               |
+|-------------------------------------------------------------|--------------------------------------------------------------|--------------------|----------|------------------------|---------------------------------------|
+| Standalone                                                  | No                                                           | Yes                | No       | 1                      |                                       |
+| [Replicated](https://redis.io/docs/management/replication/) | No (replica nodes are used e.g. to optimize read operations) | Yes                | No       | 1 master, n replicas   |                                       |
+| [Sentinel](https://redis.io/docs/management/sentinel/)      | Yes (master election in case of failure)                     | Yes                | No       | 3+                     | Needs special handling on client side |
+| [Cluster](https://redis.io/docs/management/scaling/)        | Yes (master election in case of failure)                     | No                 | Yes      | 3+ masters, n replicas | Needs special handling on client side |
+
+If you are interested in using another flavour, let us know!
```

Comment:
Would be great to specify here which one is supported by the other `redis` base.

Reply:
[Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/38/commits/04c99e377901fca1664a910ce96652267528e1c6)